src/index.html

```diff
@@ -17,6 +17,92 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="navi-bar">
```

Comment:
The class name 'navi-bar' might not best represent the meaning of the content. Consider using a name like 'nav' or 'main-nav' to better represent the element.